internal/mode/static/state/graph/tlsroute.go

```diff
@@ -86,48 +100,30 @@ func validateBackendRefTLSRoute(gtr *v1alpha2.TLSRoute,
 		Name:      string(gtr.Spec.Rules[0].BackendRefs[0].Name),
 	}
 
-	backendRef := BackendRef{
-		Valid: true,
-	}
-	var cond *conditions.Condition
-
-	if ref.Port == nil {
-		valErr := field.Required(refPath.Child("port"), "port cannot be nil")
-		backendRef.Valid = false
-		cond = helpers.GetPointer(staticConds.NewRouteBackendRefUnsupportedValue(valErr.Error()))
-
-		return cond, backendRef
-	}
-
 	svcIPFamily, svcPort, err := getIPFamilyAndPortFromRef(
 		ref,
 		svcNsName,
 		services,
 		refPath,
 	)
 
-	backendRef.ServicePort = svcPort
-	backendRef.SvcNsName = svcNsName
+	backendRef := BackendRef{
+		SvcNsName:   svcNsName,
+		ServicePort: svcPort,
+		Valid:       true,
+	}
 
 	if err != nil {
 		backendRef.Valid = false
-		cond = helpers.GetPointer(staticConds.NewRouteBackendRefRefBackendNotFound(err.Error()))
-	} else if err := verifyIPFamily(npCfg, svcIPFamily); err != nil {
```

Comment:
Why is this chunk no longer necessary?

Reply:
all these checks are done in the `validateBackendRef` function that I added on line 80